content/docs/command-reference/plots/modify.md

```diff
@@ -123,6 +123,29 @@ plots:
       title: Accuracy
 ```
 
+## Example: Make a regular output into a plot
```

Comment:
```suggestion
## Example: Convert any output into a plot
```

But I'm not sure an example is the best place for this. It seems more like a subsection of the description.

Reply:
Or add a paragraph to the description mentioning this possible, use, and linking to this example. But move the example to last (after the existing one). Whatever seems to make more sense for you — remember to look at your work-in-progress in the browser with a dev environment.

Reply:
Added a paragraph in description.